squid/tests/test_squid.py

```diff
@@ -26,3 +26,24 @@ def test_check_ok(aggregator, check, instance):
     for metric in common.EXPECTED_METRICS:
         aggregator.assert_metric("squid.cachemgr." + metric, tags=expected_tags)
     aggregator.assert_all_metrics_covered()
+
+
+@pytest.mark.usefixtures("dd_environment")
+def test_version_metadata(check, instance, datadog_agent):
+    check.check_id = 'test:123'
+
+    check.check(instance)
+
+    version = common.SQUID_SERVER_VERSION
```

Comment:
looks like this value is hardcoded. If the `latest` version no longer corresponds to `3.5.12`, this will fail. We can use mock version like in this example
https://github.com/DataDog/integrations-core/blame/5693dc717d4e41533aa693e2a87ad90f08a5985a/supervisord/tests/test_supervisord_unit.py#L247

Reply:
Updated!